examples/orderbook/create-listing-with-nextjs/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "create-listing-with-nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "playwright test"
+  },
+  "dependencies": {
+    "@biom3/react": "^0.26.1",
+    "@ethersproject/providers": "^5.7.2",
+    "@imtbl/sdk": "^1.51.2",
```

IMO "latest" might be the best option here, similar to the new passport examples, to ensure these are working with the latest deployed sdk. 
```suggestion
    "@imtbl/sdk": "latest",
```